lib/app/modules/addOrUpdateAlarm/views/maths_challenge_tile.dart

```diff
@@ -206,4 +219,12 @@ class MathsChallenge extends StatelessWidget {
       ),
     );
   }
+
+  void _presetToInitial(
```

Comment:
You should move this to the controller and not keep it in the view. Same applies to all the places you've added this.